Makefile

```diff
@@ -12,7 +13,9 @@ TEST_CMD=	tests/test_spiped.sh
 
 ### Shared code between Tarsnap projects.
 
-all:	cflags-filter.sh cpusupport-config.h posix-flags.sh
+LIBFILE=${LIB}/${LIB}.a
```

Comment:
This is the last quibble that I'm aware of: the top-level `Makefile` only works if there's a single library.

Supporting multiple LIBFILES woudn't be hard -- just move this line above the "Shared code" comment, and make it
```
LIBFILES=libspiped/libspiped.a
```

That does mean there's a bit of duplication with the earlier `LIB`, but given the extra flexibility in terms of allowing multiple libraries, it might be worth it.

(I did consider splitting `libspiped.a` into `libcperciva.a` and `libspiped.a`, where the latter only containing graceful_shutdown, dnsthread, and proto.  In the end I figured against it, but only a 30-70 split.  In our projects, I don't think it really matters either way, but an ideal build system should be able to scale up to huge projects, in which case splitting libraries would be more useful.)

Reply:
I don't think there's any point splitting the library.  If anything I would consider going in the opposite direction and making this a hard-coded `liball.a` -- partly because that would remove boilerplate of every `Makefile.BSD` specifying that yes, it uses library code, and partly to reduce the risk of someone misunderstanding the build process and thinking that "libspiped" is something which should be installed.

Reply:
If we can drop `make -f Makefile.BSD` support, then I'm all for `liball.a`.

... hmm, I suppose that you could do:
```
make -f Makefile.BSD LDADD_REQ="../../libspiped/libspiped.a -lcrypto"
```
(in the example of perftests/standalone-enc)

Reply:
Sounds like another reason to put it into `LDADD` in `Makefile.inc`?